config.json

```diff
@@ -77,5 +77,10 @@
 	"host": "localhost",
 	"port": 27018,
 	"database": "metadata"
+    },
+    "requests": {
+        "viaProxy": false,
+        "trustedProxyIPs": [],
```

Comment:
`trustedProxyCIDRs` would be more appropriate since we're not going to list ip addresses one by one.